src/Jackett.Common/Indexers/Abstract/GazelleTracker.cs

```diff
@@ -24,22 +24,23 @@ public abstract class GazelleTracker : BaseWebIndexer
     {
         protected virtual string LoginUrl => SiteLink + "login.php";
         protected virtual string APIUrl => SiteLink + "ajax.php";
-        protected virtual string DownloadUrl => SiteLink + "torrents.php?action=download&usetoken=" + (useTokens ? "1" : "0") + "&id=";
+        protected virtual string DownloadUrl => SiteLink + "torrents.php?action=download&usetoken=" + (useTokens ? "1" : "0") + "&torrent_pass=" + configData.PassKey.Value + "&id=";
```

Comment:
For those gazelle indexer that are not using a passkey, it may be better to not provide the `&torrent_pass=` parm, by checking the usePassKey boolean
```c#
        protected virtual string DownloadUrl => SiteLink + "torrents.php?action=download&usetoken=" + (useTokens ? "1" : "0") + (usePassKey ? "&torrent_pass=" + configData.PassKey.Value : "") + "&id=";
```